Sources/GRPC/Server.swift

```diff
@@ -384,6 +390,21 @@ extension Server {
     /// the need to recalculate this dictionary each time we receive an rpc.
     internal var serviceProvidersByName: [Substring: CallHandlerProvider]
 
+    #if canImport(Network)
+    /// A closure allowing to customise the `NWParameters` used when establising a connection using NIOTransportServices.
+    @available(macOS 10.14, iOS 12.0, watchOS 6.0, tvOS 12.0, *)
+    public var nwParametersConfigurator: (@Sendable (NWParameters) -> Void)? {
```

Comment:
Is this for the listener channel or for child channels?

Reply:
Good point - right now it's being used for the listener channel. Should we also expose one for the child channels?